### ISSUE TITLE
`in_seconds` doesn't work for units of months or larger

### ISSUE THREAD

**Issue by garybernhardt** [kylewlacy/timerizer#6]
Timerizer can compute the number of seconds in large numbers of days, but it can't compute the number of seconds in a month:

```
> 90.days.in_seconds
=> 7776000
> 1.months.in_seconds
=> 0
```

For comparison, doing `1.month.to_i` with ActiveRecord assumes 30 days in a month, giving `30*24*60*60` seconds, which seems reasonable.

This seems like an easy change (just add some more entries to @@in_seconds in timerizer.rb). However, I wanted to open an issue first. Is this behavior intentional?

**Comment by kylewlacy** [comment]
Hey, thanks for the bug report! No, that behavior is definitely not intentional, and if you opened a PR I'd be more than happy to review/merge it ASAP.

That said, there are a few warts in Timerizer's design that I've been wanting to address for a while (both in implementation and interface). I've held off on working on it because I haven't needed it for any personal projects recently, but I figure since people are actually using the library I might as well take the time to make some changes. Like I said, feel free to open a PR, but I figure I'll probably put some work in on Timerizer when I have some free time this weekend (and resolving this issue would be the first step of that).

**Comment by kylewlacy** [comment]
Okay, I just published a new version of Timerizer, v0.3.0 ([changelog](https://github.com/kylewlacy/timerizer/blob/8f2d2b4f1cd2cf3f560eecd3039fd30b372a70cd/CHANGELOG.md#030---2017-10-02)). `#in_seconds` is now `#to_seconds`, and it now returns a reasonable value for large durations:

```rb
90.days.to_seconds
# => 7776000
1.month.to_seconds
# => 2592000
```

There's quite a few other (breaking) changes too, but I think it's safe to close this now. Thanks again for reporting this!

**Comment by garybernhardt** [comment]
Awesome, thanks! I actually did take a look at it myself, but lost steam while digging through the implementation details and hadn't made a second attempt. Behavior looks good to me now!
